src/components/App.js

```diff
@@ -3,10 +3,12 @@ import '../styles/App.css';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
 
+import calculate from '../logic/calculate';
+
 function App() {
   return (
     <>
-      <Display result="calc result" />
+      <Display result="Hello!" />
```

Comment:
- [ ] We could bypass the linter issue by passing down the `calculate` function to display for now until you start working on it on the next milestone:

`<Display result="Hello" calculate={calculate} />`